src/components/user/user.service.ts

```diff
@@ -178,14 +192,7 @@ export class UserService {
           id,
           createdAt,
         }),
-        relation('out', '', 'email', {
-          active: true,
-          createdAt,
-        }),
-        node('email', 'EmailAddress:Property', {
-          value: input.email,
-          createdAt,
-        }),
+        ...this.emailProperty(input.email, createdAt),
```

Comment:
I think we always want the property defined even if it's null. This way we don't have to do optional matches later.
```suggestion
        ...property('email', input.email, 'user', 'email', 'EmailAddress'),
```